### ISSUE TITLE
Test & Val set up when 'weighted_ce' = 0

### ISSUE THREAD

**Issue by YH-UtMSB** [nikhil-dce/SBM-meet-GNN#1]
Hi Nikhil,

Thanks for releasing the code!

I'm a little uncertain about whether it is appropriate to use half (+) half (-) Testing and Validation set when the model is trained with the argument 'weighted_ce' set to FALSE. In applying the weigthed_CE, the (+) training samples are augmented #(-)/#(-) times so, in the Training set, the two types of samples are balanced in amount, and a balanced Testing / Validation set looks good as its samples are similarly distributed to the training samples. **Likewise, when weighted_CE is not applied, a balanced Testing / Validation set makes (+) and (-) samples disproportionate to their counterparts in the imbalanced Training set.**

I'd love to know your opinion about this issue. Meanwhile, I'm curious about the results when you set 'weighted_ce' = 0. Do they have significant differences with those when 'weighted_ce' = 1?

Looking forward to your reply. Thanks!

**Comment by nikhil-dce** [comment]
Apologies, for missing this question - 'weighted_ce' was always set to 1 in all the experiments. Unfortunately, I do not recall evaluating the model with 'weighted_ce = 0'. In my opinion, balance vs imbalance testing scenarios would depend on the use case of the model.
